script/Ciel/Day 4/04_01_CIEL04_4_2.txt

```diff
@@ -26,7 +26,7 @@ She's got me there. On a scale of normal to abnormal, my eyes are in the same ca
 [sha:0dbd051ed759abe6d08ef992699e2f9e0491b258]{
 -- Page 96, Offset 8415.
 -- ……けど、それだって<こ|・><こ|・><ま|・><で|・>じゃなかった。
-...But even my eyes aren't at this level.
+...But even %{i}my eyes%{/i} aren't at this level.
```

Comment:
just "my" in italics

Reply:
Shouldn't the italics be on _this_ here?

Reply:
which is it then?

Reply:
yeah, for whatever reason I read this as "even my eyes [in comparison to yours]" but Ross is right.